README.md

```diff
@@ -15,6 +15,8 @@ Lifecycle Configurations (LCCs) provide a mechanism to customize SageMaker Studi
 
 ### [SageMaker JupyterLab](https://docs.aws.amazon.com/sagemaker/latest/dg/studio-updated-jl.html)
 - [auto-stop-idle](jupyterlab/auto-stop-idle/) - Automatically shuts down JupyterLab applications that have been idle for a configurable time.
+- [ebs-backup-s3](jupyterlab/ebs-backup-s3) - Backs up content in a user space's EBS volume (user's home directory under `/home/sagemaker-user`) to an S3 bucket specified in a tag on the user profile. The files are stored under `s3://<bucket>/<user-profile-name>/<space-name>/<timestamp>`
```

Comment:
Can we also think about introducing a flag to schedule the sync to run every X hours? This would be useful for customers with specific RPO requirements, to which running only when the app is (re)started can be a limitation.

Reply:
Removed the script from JupyterLab, and added to `Common scripts`. Added flag to optionally schedule every 12 hours.